test/unit/iotagent-thinking-things_test.js

```diff
@@ -173,6 +185,40 @@ describe('Southbound measure reporting', function() {
         it('should return a 200OK with the configured sleep time in the core module',
             checkResponse(options, '#STACK1#5143,GPS,-1$cond1,#673495,K1,300$theCondition,'));
     });
+
+
+
+    describe('When a request arrives to the IoT Agent having Generic Configuration modules', function() {
+        var options = {
+            url: 'http://localhost:' + config.thinkingThings.port + config.thinkingThings.root + '/Receive',
+            method: 'POST',
+            form: {
+                cadena: '#STACK1#0,GC,conf,33,600$,#1,GC,conf2,123,600$,#673495,K1,2500$theCondition,'
```

Comment:
Do you refer to the payload? In that case I think the developer that is looking at the tests of the agent should have the protocol documentation at hand, and use it to understand the payload, instead of having to explain it in every test. The english description is in the `describe()` above: "... having Generic Configuration modules", that is, `GC` instructions, as will be displayed in the documentation. The next comment you will make is: there is not such thing as "the documented protocol". And you are right. That's why I created an issue to write the documentation. I expect to have it today, but as part of another PR (as the whole protocol needs more documentation, not only the Generic parts).